File: php-laraval-app/app/Models/LaundryOrderHistory.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\SoftDeletes;

class LaundryOrderHistory extends Model
{
    use HasFactory;
    use SoftDeletes;

    /**
     * Indicates if the model should be timestamped.
     *
     * @var bool
     */
    public $timestamps = true;

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'laundry_order_id',
        'type',
        'note',
        'causer_id',
    ];

    /**
     * The attributes that should be cast.
     *
     * @var array
     */
    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
        'deleted_at' => 'datetime',
    ];

    public function laundryOrder(): BelongsTo
    {
        return $this->belongsTo(LaundryOrder::class)->withTrashed();
    }

    public function causer(): BelongsTo
    {
        return $this->belongsTo(User::class)->withTrashed();
    }
}

File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/LaundryOrderHistory.cs
namespace Downstairs.Infrastructure.Persistence.Models;

public partial class LaundryOrderHistory
{
    public long Id { get; set; }

    public long LaundryOrderId { get; set; }

    public string Type { get; set; } = null!;

    public string Note { get; set; } = null!;

    public long CauserId { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public DateTime? DeletedAt { get; set; }

    public virtual User Causer { get; set; } = null!;

    public virtual LaundryOrder LaundryOrder { get; set; } = null!;
}